README.md

```diff
@@ -1,24 +1,125 @@
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+> Developed using Ruby on Rails.
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **Register and login features**
+- **Database where you can add categories and transactions**
+- **Implemented design from the sneak peek wireframe**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [PocketPal](https://pocketpal.onrender.com/)
+- [Video](https://drive.google.com/file/d/1wXQctD8fxHFhkzR3kd55fuuhozXjljYA/view?usp=sharing)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have ruby on rails installed in your computer.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Installation
+
+Install this project with:
+
+```
+- bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+- rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+- rspec
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Erick Hans Martinez**
+
+- GitHub: [@erickhansx](https://github.com/erickhansx)
+- Twitter: [@TheErickHans](https://twitter.com/TheErickHans)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> These are some of the future features we want to add.
+
+- [ ] **_Add a remove button for categories_**
+- [ ] **_Add a remove button for each transaction_**
+- [ ] **_Add a all transactions page_**
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to the code reviewers and my partners who are helping all the time.
+Based on the original idea of [Gregoire Vella on Behance](https://www.behance.net/gregoirevella).
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job @erickhansx 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.

Reply:
Thank you so much for your review and observation my friend I will do so!